### DISCUSSION TITLE
blog/2023-10-02-network/

### DISCUSSION THREAD

**Discussion by giscus** [Tom910/personal-blog#12] [General]
# blog/2023-10-02-network/

Dive into the intricacies of network infrastructure, exploring topics from the fundamentals of DNS to the complexities of modern data center designs. Whether you're a developer or a network engineer, this article serves as a comprehensive guide to understanding how data travels across the internet and how modern data centers operate at scale.

https://amarchenko.dev/blog/2023-10-02-network/

**Comment by vinu76jsr** [comment]
Hi Andrei, 

this was a really great read, I especially love these kind of longform articles. thanks for taking time to write this.
